### PR TITLE
feat: added new files icon #173

### PR DESCRIPTION
**Pull request by bloodmonk7**
With kind assistance of @jairochaves2, I added the three left files icon as mention in issue #173 .

### CODE REVIEW

File: projects/ion/src/lib/icon/svgs/iconsText/doc-check.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jairochaves2) [inline projects/ion/src/lib/icon/svgs/iconsText/doc-check.ts:1]: Please remove the svg tags in the files, it is already added in the icon component

Comment (bloodmonk7) [reply projects/ion/src/lib/icon/svgs/iconsText/doc-check.ts]: Removed the svg tags from all 3 files @jairochaves2

**Comment by iurynogueira** [comment]
Nice support @jairochaves2, thanks! 😃

**Review by iurynogueira — APPROVED** [review]

**Comment by iurynogueira** [comment]
I am checking pipeline secrets, github actions not found the secrets.

**Review by jairochaves2 — APPROVED** [review]

**Comment by bloodmonk7** [comment]
Am I required to do something, or it will get merged?

**Comment by iurynogueira** [comment]
> Am I required to do something, or it will get merged?

merged! Thanks for your contribute @bloodmonk7

**Pull request merged**